go.mod

```diff
@@ -11,7 +11,9 @@ require (
 	github.com/avast/retry-go v2.6.1+incompatible
 	github.com/aws/aws-sdk-go v1.36.15
 	github.com/cenkalti/backoff/v3 v3.2.2 // indirect
-	github.com/cockroachdb/pebble v0.0.0-20201130172119-f19faf8529d6
+	github.com/cockroachdb/errors v1.8.2 // indirect
+	github.com/cockroachdb/pebble v0.0.0-20210113182756-a53f229c8c9c
```

Comment:
I'm assuming this is inentional?

Reply:
Of course!  You didn't seriously expect to be able to write SSTable metadata with `v0.0.0-20201130172119-f19faf8529d6`?!